src/ui/generic/appLoading.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react'
+import { connect } from 'react-redux'
+import { RootState } from 'src/reducers'
+
+import { LoadingScreen } from './loading'
+import { LoadingSpinner } from './loadingSpinner'
+
+interface Props extends ReturnType<typeof mapStateToProps> {}
+
+export const AppLoadingContainer: React.SFC<Props> = props =>
```

Comment:
Isn't `SFC<Props>` deprecated? or are we using an older version of react?

Reply:
In VSCode if I hover over SFC, it does indeed say it is deprecated, and that we should be using `React.FunctionComponent<Props>` instead.